core/src/main/java/com/alibaba/nacos/core/distributed/raft/JRaftProtocol.java

```diff
@@ -123,36 +124,39 @@ public void init(RaftConfig config) {
             
             // There is only one consumer to ensure that the internal consumption
             // is sequential and there is no concurrent competition
-            NotifyCenter.registerSubscribe(new Subscribe<RaftEvent>() {
-                @Override
-                public void onEvent(RaftEvent event) {
-                    Loggers.RAFT.info("This Raft event changes : {}", event);
-                    final String groupId = event.getGroupId();
-                    Map<String, Map<String, Object>> value = new HashMap<>();
-                    Map<String, Object> properties = new HashMap<>();
-                    final String leader = event.getLeader();
-                    final Long term = event.getTerm();
-                    final List<String> raftClusterInfo = event.getRaftClusterInfo();
+            try {
+                NotifyCenter.registerSubscriber(new Subscriber<RaftEvent>() {
```

Comment:
And registered listeners should have no abnormal perception，Readjust the interface inside the NotifyCenter and adjust the BiFunction so that no exceptions are thrown

Reply:
The BiFunction should throws NacosRuntimeException.Because the code of clazz.newInstance(), will throws NacosRuntimeException.And I will adjust this code in next pr' commit.